LibSource/ComplexFloatArray.h

```diff
@@ -75,6 +75,29 @@ struct ComplexFloat {
     im = magnitude*sinf(phase);
   }
 
+  float getMagnitudeSquared() {
```

Comment:
you don't like `  float getMagnitudeSquared() const` on line 35?

Reply:
I like it so much that I've doubled it! Ok, it probably was done by mistake. Also, I think I should mark stuff like getComplexConjugate / getDotProduct as const.